web/beacon-app/src/application/api/ApiService.ts

```diff
@@ -15,30 +15,22 @@ axiosInstance.defaults.withCredentials = true;
 // intercept request and check if token has expired or not
 axiosInstance.interceptors.request.use(
   async (config: any) => {
-    const token = getCookie('bc_atk');
-    const csrfToken = getCookie('csrf_token');
-    const decodedToken = token && decodeToken(token);
-    if (decodedToken) {
-      const { exp } = decodedToken;
-      const now = new Date().getTime() / 1000;
-      if (exp < now) {
-        //console.log('token expired');
-        // refresh token
-
-        // refreshToken();
-        //const accessToken = getCookie('bc_atk');
-        // const { data } = await QuarterDeckAuth.refreshToken(refreshToken);
-        // setCookie('bc_atk', data.access_token, { expires: data.expires_in });
-        // setCookie('bc_rtk', data.refresh_token, { expires: data.expires_in });
-        // setCookie('csrf_token', data.csrf_token, { expires: data.expires_in });
-        //config.headers.Authorization = `Bearer ${accessToken}`;
-        return config;
+    // since the server set the cookie with httpOnly flag
```

Comment:
Is the sentence below similar to the comment on lines `18-19`?

`The cookie is automatically set by the browser because the server sets cookies with an httpOnly flag.`

Reply:
Great. i updated the comment message if it wasn't clear.